src/index.js

```diff
@@ -603,10 +603,12 @@ const sortByOrder = function (candidates) {
     .concat(regularTabbables);
 };
 
+const byDocumentOrder = (a, b) => (a.compareDocumentPosition(b) & 4 ? -1 : 1);
```

Comment:
Could we use the constant property instead of `4` when testing the bitmask? I think that would clarify the intent of the test.

```suggestion
const byDocumentOrder = (a, b) => (a.compareDocumentPosition(b) & Node.DOCUMENT_POSITION_FOLLOWING ? -1 : 1);
```

Reply:
Of course!